src/main/java/group1/mips_simulator/components/instructionExecution/ExecutionResult.java

```diff
@@ -0,0 +1,22 @@
+package group1.mips_simulator.components.instructionExecution;
+
+public class ExecutionResult {
```

Comment:
i see, interesting architecture, packaging all possible results into one class, i like it